griptape/tools/google_drive/tool.py

```diff
@@ -173,13 +176,16 @@ def download_files(self, params: dict) -> ListArtifact | ErrorArtifact:
                 else:
                     logging.error(f"Could not find file: {path}")
     
-            except HttpError as e:
-                logging.error(e)
-    
-            except MalformedError:
-                logging.error(f"MalformedError occurred while downloading file '{path}' from Google Drive")
-    
-        return ListArtifact(downloaded_files)
+            return ListArtifact(downloaded_files)
+        except HttpError as e:
+            logging.error(e)
```

Comment:
@vasinov should we remove the `logging.error`s? None of the other tools really have it, and seems redundant since we're already returning an `ErrorArtifact` which could take care of the logging.

Reply:
Agreed.